evmcore/tx_cacher.go

```
@@ -62,7 +63,9 @@ func newTxSenderCacher(threads int) *txSenderCacher {
 func (cacher *txSenderCacher) cache() {
 	for task := range cacher.tasks {
 		for i := 0; i < len(task.txs); i += task.inc {
-			types.Sender(task.signer, task.txs[i])
+			if _, err := types.Sender(task.signer, task.txs[i]); err != nil {
```

This may be quite verbose and not very productive. 
I think it would be OK just to ignore this error (with a comment). Since the intended use is to cache values, if any to be cached. 
wdyt?

---

I agree, this might be a case where we can reasonably justify to ignore the error.